src/makeRobot.js

```diff
@@ -38,7 +38,65 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const evacuateCoordX = 1400;
+  const evacuateCoordY = 500;
+
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${name}, chip version: ${version}, wheels: ${wheels}`;
+    },
+
+    get location() {
+      return `${name}: x=${this.coords.x}, y=${this.coords.y}`;
+    },
+
+    goBack(num = 1) {
+      if (num >= 0) {
```

Comment:
do you really need to add 0? fix it everywhere
```suggestion
      if (num > 0) {
```